entity-framework/core/querying/related-data/eager.md

```diff
@@ -120,22 +120,44 @@ public class School
 }
 ```
 
-Contents of `School` navigation of all People who are Students can be eagerly loaded using a number of patterns:
+Contents of `School` navigation of all People who are Students can be eagerly loaded using many patterns:
 
-* using cast
+* Using cast
 
   ```csharp
   context.People.Include(person => ((Student)person).School).ToList()
   ```
 
-* using `as` operator
+* Using `as` operator
 
   ```csharp
   context.People.Include(person => (person as Student).School).ToList()
   ```
 
-* using overload of `Include` that takes parameter of type `string`
+* Using overload of `Include` that takes parameter of type `string`
 
   ```csharp
   context.People.Include("School").ToList()
   ```
+
+## Model configuration for auto-including navigations
+
+> [!NOTE]
+> This feature was introduced in EF Core 6.0.
+
+You can configure a navigation in the model to be included every time the entity is loaded from the database using `AutoInclude` method. It has same effect as specifying `Include` with the navigation in every query where the entity type is returned in the results. Following example shows how to configure a navigation to be automatically included.
```

Comment:
Link to API docs

Reply:
Filed https://github.com/dotnet/EntityFramework.Docs/issues/3551

Will update once we publish API Docs.